library/src/main/res/values/library_rxlifecycle_strings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <string name="define_rxlifecycle"></string>
```

Comment:
should be define_int_* as like here:
https://github.com/WoogieNoogie/AboutLibraries/blob/19167939b4ffccb09ada141d1dd46904e96d9d53/library/src/main/res/values/library_sugarorm_strings.xml#L3